sql/analyze_test.go

```diff
@@ -31,10 +31,14 @@ func testTableDesc() *TableDescriptor {
 		ID:   1000,
 		Columns: []ColumnDescriptor{
 			{Name: "a", Type: ColumnType{Kind: ColumnType_INT}},
-			{Name: "b", Type: ColumnType{Kind: ColumnType_INT}},
-			{Name: "c", Type: ColumnType{Kind: ColumnType_INT}},
-			{Name: "d", Type: ColumnType{Kind: ColumnType_INT}},
-			{Name: "e", Type: ColumnType{Kind: ColumnType_INT}},
+			{Name: "b", Type: ColumnType{Kind: ColumnType_BOOL}},
```

Comment:
Ah, I see you've changed those tests to use column `i`. You could have avoided touching those tests by leaving column `b` with type INT. Oh well, doesn't matter now.